src/components/Todo/List.jsx

```diff
@@ -1,14 +1,14 @@
 import Item from "./Item";
 
 function List( props ) {
-  const { todoList } = props
+  const { todoList, onRemoveTodo } = props
 
   return (
     <div>
       {todoList && todoList.map(item => (
```

Comment:
nit: if you set your state correctly, to be consistently an array, whether empty or not. then you don't need this `todoList &&` bit. since the map portion won't fire if you have an empty array, and it won't throw errors